applications/nrf_desktop/src/modules/ble_conn_params.c

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <bluetooth/conn.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/gatt_dm.h>
+
+#ifdef CONFIG_BT_LL_NRFXLIB
+#include "ble_controller_hci_vs.h"
+#endif /* CONFIG_BT_LL_NRFXLIB */
+
+#define MODULE ble_conn_params
+#include "module_state_event.h"
+#include "ble_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_CONN_PARAMS_LOG_LEVEL);
+
+#define CONN_INTERVAL_LLPM_US		1000   /* 1 ms */
+#define CONN_INTERVAL_LLPM_REG		0x0d01 /* 1 ms */
+#define CONN_INTERVAL_BLE_REG		0x0006 /* 7.5 ms */
+#define CONN_LATENCY_LOW		0
+#define CONN_LATENCY_DEFAULT		99
+#define CONN_SUPERVISION_TIMEOUT	400
+
+#define CONN_PARAMS_ERROR_TIMEOUT	K_MSEC(5)
+
+struct connected_peer {
+	struct bt_conn *conn;
```

Comment:
const

Reply:
done

Reply:
`bt_conn_le_param_update` takes `struct bt_conn *conn` as an argument.
The connection object is modified there, so we should not use the const pointer in the module.